### ISSUE TITLE
Why generator loss in vae model not plus the ae's cost

### ISSUE THREAD

**Issue by seigercom** [Ruiqi-Hu/ARGA#6]
Hi, it's a nice work and thanks for sharing the code.
I have a puzzle, in arga_vae model, when you define the OptimizerVAE in optimizer.py, the generator_loss doesn't plus the auto-encoder cost
`self.generator_loss = tf.reduce_mean(
            tf.nn.sigmoid_cross_entropy_with_logits(labels=tf.ones_like(d_fake), logits=d_fake))`
rather than
`self.generator_loss = tf.reduce_mean(
            tf.nn.sigmoid_cross_entropy_with_logits(labels=tf.ones_like(d_fake), logits=d_fake))+self.cost`.
Can you please tell me the reason to do this?
Thanks a lot.

**Comment by DreamerDW** [comment]
Hello, what is the generator_loss?
